WC-Local-Pickup/woocommerce-local-pickup-time
# Provides the Ability to Deploy to WordPress.org via a GitHub Release

### All Submissions:






### Changes proposed in this Pull Request:



Closes #115

- Initial Updates & Changes in Preparation for New GH Action
- Fixes Version Increase Scripts
- Fixes Version Increase Script Without i18n
- README & Language File Version Updates
- Updates to GitHub Actions & Prep for Releases


### How to test the changes in this Pull Request:

1. Prepare a new release branch.
2. Open up a PR from the release branch to `main`.
3. Merge the PR into `main`.
4. Create a GitHub Release/tag for the new version.
5. Observer the new Release GitHub Action workflow should run and deploy to WordPress.org.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- New GitHub Action and local build preparation changes to support WordPress.org releases.
